airbyte-integrations/connectors/source-first-connector/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM scratch
+
+## TODO Add your dockerfile instructions here
+
+FROM airbyte/source-first-connector:dev
+
+WORKDIR /airbyte
+
+ENV APPLICATION source-first-connector
+
+
+
+# Airbyte's build system uses these labels to know what to name and tag the docker images produced by this Dockerfile.
+LABEL io.airbyte.name=airbyte/source-first-connector
+LABEL io.airbyte.version=0.1.0
```

Comment:
all files should have a newline character, in the end, this is why GitHub shows you this 🛑   sign